tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.34.1-stretch
+FROM rust:1.37.0-buster
```

Comment:
Without this change, `docker build` still works.
Should I remove all changes related with rust?

Reply:
NOTE: we decided to update the rust image with this PR